packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -134,8 +134,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     // (undocumented)
     readonly options: Record<string | number, any>;
     orderSequentially<T>(callback: () => T): T;
-    // (undocumented)
-    process(messageArg: ISequencedDocumentMessage, local: boolean): void;
+    process({ ...messageArg }: ISequencedDocumentMessage, local: boolean): void;
```

Comment:
@markfields I hope this is not a public api change? We are not changing the signature of the process() method, so it should be ok?

Reply:
Yep agreed